src/main/java/mate/academy/util/UtilConnection.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class UtilConnection {
+    private static final String URL = "jdbc:mysql://localhost:3306/book";
+    private static final Properties PROPERTIES;
+    private static final String DRIVER_CLASS_NAME = "com.mysql.cj.jdbc.Driver";
+    private static final String LOGIN = "root";
+    private static final String PASSWORD = "root";
+
+    static {
+        PROPERTIES = new Properties();
+        PROPERTIES.put("user", LOGIN);
+        PROPERTIES.put("password", PASSWORD);
+    }
```

Comment:
I would suggest to make 'user' and 'password' literals constants rather than it`s values, because values could change dynamically, but words 'user' and 'password' will remain same